messages/src/command.rs

```diff
@@ -3,17 +3,38 @@ use crate::util::{ascii_to_bool, bool_to_ascii};
 use crate::WriteCompat;
 use crate::{ProtocolError, Side};
 use core::convert::TryInto;
+use core::fmt::{self, Display, Formatter};
 use core::mem::size_of;
 use core::ops::RangeInclusive;
 use nb::Error::{Other, WouldBlock};
 
+/// A note to play on the buzzer.
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq)]
+pub struct Note {
+    /// The frequency in Hertz, or `None` for silence.
+    pub frequency: Option<u32>,
```

Comment:
`Option<NonZero<u32>>` uses fewer bytes and encodes what you mean better

Reply:
Aha, I was wondering about something like that. Done.